.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -7,6 +7,17 @@ body:
     attributes:
       value: '# This form is for Renovate maintainers only.'
 
+  - type: checkboxes
+    id: are-you-a-maintainer
+    attributes:
+      label: Are you a maintainer?
+      description: |
+        We only accept issues from Renovate maintainers. If you are not a maintainer, please create a Discussion instead.
+      options:
+        - label: I am a maintainer
+    validations:
+      required: true
```

Comment:
I suspect I'm doing it wrong. There's no `*` next to the label text, when I preview the forms on my branch.

The next field `How are you running Renovate?` does show the `*` in the preview. 😜